contrib/spec/podman.spec.in

```diff
@@ -361,6 +360,7 @@ providing packages with %{import_path} prefix.
 %prep
 %autosetup -Sgit -n podman-%{shortcommit}
 sed -i '/\/bin\/env/d' completions/bash/%{name}
+mv pkg/hooks/README.md pkg/hooks/README-hooks.md
```

Comment:
I'd removed this in #772, which moved all critical information from `pkg/hooks/README-hooks.md` into `oci-hooks.5`, which should be installed via [this][1] and [this][2].  Does that not work for some reason?

[1]: https://github.com/projectatomic/libpod/blob/4704c138ae7f3c98c3fddba2bc442d61f4eb1f28/contrib/spec/podman.spec.in#L385
[2]: https://github.com/projectatomic/libpod/blob/4704c138ae7f3c98c3fddba2bc442d61f4eb1f28/contrib/spec/podman.spec.in#L464